presto-main/src/main/java/com/facebook/presto/operator/scalar/IntervalFunctions.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I am still not sure about the best place to add this new function. We mention various interval functions in the official Presto docs in the "Date and Time Functions and Operators" section, so maybe we should add this function to `DateTimeFunctions` (and move the test to `TestDateTimeFunctions`).